lib/jsdoc/util/logger.js

```diff
@@ -130,34 +141,26 @@ function addPrefix(args, prefix) {
     return updatedArgs || args;
 }
 
-// TODO: document events
-function wrapLogFunction(name, func) {
-    var eventName = 'logger:' + name;
-    var upperCaseName = name.toUpperCase();
-    var level = LEVELS[upperCaseName];
-    var prefix = PREFIXES[upperCaseName];
-
-    return function() {
-        var loggerArgs;
 
-        var args = Array.prototype.slice.call(arguments, 0);
-
-        if (logLevel >= level) {
-            loggerArgs = addPrefix(args, prefix);
-            func.apply(null, loggerArgs);
-        }
+var Logger = function () {
+    this.handlers = {};
```

Comment:
Any reason why we can't just attach event listeners to handle events, as we currently do? By tracking the handlers here, it seems like we're duplicating work that the event emitter is already doing. (And since you can still attach listeners to the emitter directly, there's no guarantee that the list of "handlers" is complete.)